requirements-test.txt

```diff
@@ -16,3 +16,6 @@ pytest-cov == 2.10.0
 pytest-mpi == 0.4
 pytest-timeout == 1.4.2
 timm >= 0.4.5
+
+# Dependency for factory
+hydra-core >= 1.1
```

Comment:
nit @jieru-hu, but duplicate right ?

Reply:
hmm i don't think this is a duplicate - since i only added the hydra-core dependency to the examples.

Reply:
[see](https://github.com/facebookresearch/xformers/pull/93/files/247dfff2539c54806c679984bee430c8c1424eca?authenticity_token=M1RDgb83nCa5pGcC4hPE1U6MuK3k54Q%2FbV2xDFz935cqAxbqab3fg2PY3LCYwlkbeZSqAsqnQDLQJ%2BsvShcB8g%3D%3D&file-filters%5B%5D=.json&file-filters%5B%5D=.py&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=dotfile#r760626101), right ?

Reply:
woop you're right @jieru-hu , I missed the fact that the other requirements file was under example.. all good !